src/pyscipopt/scip.pxd

```diff
@@ -794,6 +817,7 @@ cdef extern from "scip/scip.h":
     SCIP_SOL* SCIPgetBestSol(SCIP* scip)
     SCIP_Real SCIPgetSolVal(SCIP* scip, SCIP_SOL* sol, SCIP_VAR* var)
     SCIP_RETCODE SCIPwriteVarName(SCIP* scip, FILE* outfile, SCIP_VAR* var, SCIP_Bool vartype)
+    SCIP_RETCODE SCIPgetTransformedVar(SCIP* scip, SCIP_VAR* var, SCIP_VAR** transvar)
```

Comment:
Could you explain where this is used for your intended functionality?

Reply:
I have not used it yet. Sorry. :)